lib/transformers/excel.js

```diff
@@ -17,26 +17,24 @@
 
 const { Transform } = require('stream')
 const XLSXTransformStream = require('xlsx-write-stream')
+const { formatDataForColumnExport } = require('./utils')
+
 
 module.exports = () => ({
-  stringifier: () => {
+  stringifier: (fieldNames) => {
```

Comment:
```suggestion
  stringifier: ({ headers: fieldNames }) => {
```

Would it be more "future proof" using an object containing the property `headers` as parameter (acting as an _options_)?  
In this manner, in case we may need to add further parameters to the other stringifiers, we don't have to deal with options order.

What do you think?

Reply:
Agree, I would not call it "headers" since it may cause confusion with http headers; I'm suggesting the use of `fields` which is a name that's already in use for this kind of items and still makes sense for such file formats.